build.sbt

```diff
@@ -60,7 +60,7 @@ def commonBaseSettings: Seq[Setting[_]] = Def.settings(
     HeaderLicense.Custom(
       """|sbt
        |Copyright 2023, Scala center
-       |Copyright 2011 - 2022, Lightbend, Inc.
+       |Copyright 2011 - 2022, Lightbend, Inc. dba Akka
```

Comment:
Since during 2011 - 2022, Lightbend wasn't doing business as (dba) Akka, I don't think we need to add this into the header unless we're asked?

Reply:
Ah I forgot about that! I guess we shall close the PR for now, and if Seth deems it necessary we can reopen the PR